css/top-bar.css

```diff
@@ -52,4 +54,11 @@
 	font-weight: 600;
 	height: 80px;
 	margin-top: -40px;
+}
+
+#top-bar .content-container {
+    background: white;
```

Comment:
might be worth specifying the property as background-color? & also might be worth using RGB or hex values (even though it's only white)?